app/views/list.html

```diff
@@ -12,9 +12,12 @@ <h4 translate translate-cloak>{{ctrl.data.title}}</h4>
     </div>
 
     <div class="ad-page-content">
-        <button ng-repeat="object in ctrl.data.objects track by $index" type="button" class="btn {{object.buttonType}} ad-header-button ng-class:[{'disabled':object.publicity === ctrl.PUBLICITY.SECRET}]" ng-click="ctrl.select(object)">{{object.title}}</button>
+        <div class="ad-page-container">
+            <button ng-repeat="object in ctrl.data.objects track by $index" type="button" class="btn btn-info btn-lg btn-block wrap-button-text db-btn-default {{object.buttonType}} ng-class:[{'disabled':object.publicity === ctrl.PUBLICITY.SECRET}]"
```

Comment:
As discussed, let's see which property should be used for disabling an item. It might not be `publicity`.